File: Python/PRAK603-2210817210005-MFithra_Siroja.py
line1,line2= map(int, input().split())
if(line1==line2):
    baris1= list(map(int,input().split()))
    baris2= list(map(int,input().split()))
    for i in range(line1):
        print(baris1[i]*baris2[i], end= ' ')
else:
    print('Jumlah tidak sama')

File: C/PRAK603-2210817210005-MFithra_Siroja.c
#include<stdio.h>

int main(void){
int line1,line2, input,matriks;
scanf("%d %d", &line1, &line2);
int matriks1[line1], matriks2[line2];
    if(line1==line2){
        for(int i=0;i<line1;i++){
        scanf("%d",&input);
        matriks1[i]=input;
        }
        for(int i=0;i<line2;i++){
            scanf("%d",&input);
            matriks2[i]=input;
        }
        for(int i=0;i<line1;i++){
            printf("%d ",matriks1[i]*matriks2[i]);}}   
    else {
       printf("Jumlah tidak sama") ;}
}